elements/standard/ratedunqueue.cc

```diff
@@ -55,6 +55,7 @@ RatedUnqueue::configure_helper(TokenBucket *tb, bool is_bandwidth, Element *elt,
     else
 	args.read_mp("RATE", r);
     if (args.read("BURST_DURATION", SecondsArg(3), dur_msec).read_status(dur_specified)
+	.read_or_set("BURST", ((RatedUnqueue*)elt)->_burst, 32)
```

Comment:
Here. `RatedUnqueue::configure_helper` is called by elements, which are not subclasses of `RatedUnqueue`: `RatedSplitter` and `BandwidthRatedSplitter`.